source/utilities.tex

```diff
@@ -15474,3 +15474,131 @@
 Otherwise, \tcode{endian::native} is not equal
 to either \tcode{endian::big} or \tcode{endian::little}.
 \end{itemdescr}
+
+\rSec1[stdbit.h.syn]{Header \tcode{<stdbit.h>} synopsis}
```

Comment:
@jensmaurer  Should we add this header to some global table at the beginning of the clause?

Reply:
I already add this to [c.headers], above. I'm not seeing another suitable global table.